packages/e2e/alonzo-private-network/byron/genesis.json

```
@@ -0,0 +1,47 @@
+{
+  "bootStakeholders": {
+    "5c4c7ba55cad968e23ef7182f31e101853936c8e5198ea0f7e002c2f": 1,
+    "dcde51d921e3cbbb7640046f4014b28061237c2e25ea519e46b9a7d8": 1
+  },
+  "heavyDelegation": {
+    "5c4c7ba55cad968e23ef7182f31e101853936c8e5198ea0f7e002c2f": {
+      "omega": 0,
+      "issuerPk": "YzNJfnWOZYPZMntflQPBP8dobm+ly4OwZa8GhKhFlrbJc3l4rHjvNBDoWSjD5ALZy5153RICYqxLj5cavv4sEA==",
+      "delegatePk": "w8w8uD4U51sd7kGpQnJBHdqILA+Ix5JH+hSnvmok7C81GcvqVJTICQLhBih5ytNBZvGAYwxyP5v5rgHVDsdGQQ==",
+      "cert": "7b04c0a57fa298f4ef234e01e6cc6403d2cfc4b36f2fee0965d401d300f6108855b8349bafbbc874a64214e4da3a42e081eb2c5fb5796177c0bd297b3dddef03"
+    },
+    "dcde51d921e3cbbb7640046f4014b28061237c2e25ea519e46b9a7d8": {
+      "omega": 0,
+      "issuerPk": "zlf/JSnxbkH1JjRRDYk5FzSQrPremLk4n90PaG7vFAv8zHRke9V4P3cCuiCzxkq8v8iKqH49S6rJHgOeLgQFrg==",
+      "delegatePk": "bGcD3YTBOJK6tMo5aGC5GDAzWnVg9KOpWxkkzM3NO6gI3QKC1tv7IieyLnBIpUEAHEpiTDPdgyrjSEO9t13WOw==",
+      "cert": "3e56dfd3f8d95834067bc61cb31dd87173c17200029fdf258245581214905d6c4189fe92c60c0925f993f2226fe27ddb81a8b9fee4b8c8086fe0dd17305a6e0c"
+    }
+  },
+  "startTime": 1655468379,
+  "nonAvvmBalances": {
+    "2657WMsDfac64sb5zNRHipRkGUU7TJtXn8gYyAfjSWjUMMsKS4Whj69EiVppoacmT": "5010000000",
+    "2657WMsDfac6K4SWtXzdYYSpHKzfMCn56s8c9eii2BECPX1vX6qvqDi6nwiMvY8TM": "5010000000"
+  },
+  "blockVersionData": {
+    "scriptVersion": 0,
+    "slotDuration": "20000",
+    "maxBlockSize": "2000000",
+    "maxHeaderSize": "2000000",
+    "maxTxSize": "4096",
+    "maxProposalSize": "700",
+    "mpcThd": "20000000000000",
+    "heavyDelThd": "300000000000",
+    "updateVoteThd": "1000000000000",
+    "updateProposalThd": "100000000000000",
+    "updateImplicit": "10000",
+    "softforkRule": {
+      "initThd": "900000000000000",
+      "minThd": "600000000000000",
+      "thdDecrement": "50000000000000"
+    },
+    "txFeePolicy": { "summand": "155381000000000", "multiplier": "43946000000" },
+    "unlockStakeEpoch": "18446744073709551615"
+  },
+  "protocolConsts": { "k": 36, "protocolMagic": 42 },
```

```suggestion
  "protocolConsts": { "k": 36, "protocolMagic": 888 },
```
Let's choose a new magic to ensure there's no confusion. 42 has been used in a public test network before

---

fixed here https://github.com/input-output-hk/cardano-js-sdk/pull/305/commits/dd26e29ab8c2753f3d8d02966f9ac009ac699344